pom.xml

```diff
@@ -78,6 +78,13 @@
         <version>4.12</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+	<groupId>imagej</groupId>
+        <artifactId>ij</artifactId>
+	<version>1.8.0_45</version>
+	<scope>system</scope>
+	<systemPath>${IMAGEJ}/ImageJ/ij.jar</systemPath>
```

Comment:
This won't work; it depends on a local artifact you downloaded. Is this not in Maven Central?